### ISSUE TITLE
UDF location for builds not working

### ISSUE THREAD

**Issue by ThomasPRZilliox** [kleinsimon/LV-WebView2#7]
Dear Simon,

TL;DR: changing the location of the UDF directory works in LV21-64bit but not in the build.

As mentioned in the [webview2 documentation](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder?tabs=dotnet#the-default-udf-location),  by default a UDF directory is created in the application directory with the name the executable (exe) path of your app + .WebView2, for example myapp.exe.webView2.

I made a very basic script and I was able to change that location on the LabVIEW IDE (LV21 64bit) but not on the build... As follow is a screenshot of my VI used to build the app:

![image](https://github.com/kleinsimon/LV-WebView2/assets/106389298/ce4409e9-ab12-45dc-80bf-30f9cc0f9bf2)

The issue is that on some computers, my app does not necessarily have the appropriate R/W accesses when installed, i.e. in Program Files, resulting in this error:

![webview](https://github.com/kleinsimon/LV-WebView2/assets/106389298/d0dc97be-b3c1-4836-9158-ddd03d4e2627)

### Workaround

**However** a work around, which is perfectly fine in my case is too change the file and folder attributes in the installer setting:

![image](https://github.com/kleinsimon/LV-WebView2/assets/106389298/cd286156-69e5-4c83-849a-847ff8499b00)
